### PR TITLE
feat: show address on form

### PR DESCRIPTION
**Pull request by MNAlfiansyah**
## Changes

- adding address view on form

## Evidence
title: Address of form
project: Jabar Super Apps
participants: @rayfajars @ristianaditya @johndy2742 @MNAlfiansyah

### CODE REVIEW

File: dashboard/components/Object/Form/index.vue [hunk 1]
```diff
@@ -231,58 +234,58 @@
...
+
+  const idData = ref(useRoute().query?.id)
+  const dataUrlImage = ref('')
+  const dataUrlImageMultiple = ref([])
+  const showDialogAddress = ref(false)
+  const address = ref({})
+  const dataOrganisasi = ref([])
+  const isFormDisabled = ref(true)
+  const selectedOrganisasi = ref({})
+  const address_all = ref('')
```
Comment (Ibwedagama) [inline dashboard/components/Object/Form/index.vue:287]: according to our JS Naming Convention, all variable naming should be `camelCase` @MNAlfiansyah 

https://wiki.digitalservice.id/doc/js-naming-conventions-gRnYoPmhnd

File: dashboard/components/Object/Form/index.vue [hunk 2]
```diff
@@ -105,13 +99,22 @@
...
                   />
                 </div>
               </div>
+              <div class="col-span-2">
+                <div class="mt-3">
+                  <BaseTextareaInput
+                    name="address_all"
+                    :value="address_all"
+                    :disabled="true"
+                  />
```
Comment (Ibwedagama) [inline dashboard/components/Object/Form/index.vue:115]: hello @MNAlfiansyah usually when I'm using a Base Component like Input or Text Area, I want to use v-model

can I use v-model with `BaseTextAreaInput`?

File: dashboard/components/Organization/Form/index.vue [hunk 3]
```diff
@@ -65,22 +60,30 @@
...
               </div>
-
-              <div class="col-span-6 mt-5">
+              <div class="col-span-2">
+                <div class="mt-3">
+                  <BaseTextareaInput
+                    name="address_all"
+                    :value="address_all"
+                    :disabled="true"
+                  />
```
[the remark this answers is not in the document]
Comment (MNAlfiansyah) [reply dashboard/components/Organization/Form/index.vue:77]: yes kang, this one just showing the address for view only

File: dashboard/components/Object/Form/index.vue (hunk 1 above)
Comment (MNAlfiansyah) [reply dashboard/components/Object/Form/index.vue]: sorry kang, it fixed [6e03e84](https://github.com/jabardigitalservice/event-management-system/pull/110/commits/6e03e84e82be5c785aef903a6ddffb97bcc330fd)

**Review by marsellavaleria19 — APPROVED** [review]

**Pull request merged**